contrib/cctz-cmake/CMakeLists.txt

```diff
@@ -82,7 +82,7 @@ if (NOT EXTERNAL_CCTZ_LIBRARY_FOUND OR NOT EXTERNAL_CCTZ_LIBRARY_WORKS)
 
         # each file/symlink in that dir (except of tab and localtime) store the info about timezone
```

Comment:
Why we don't need to store links anymore? Also need to fix a comment.

Reply:
There is no symlinks *in CCTZ repository* from where we get time zones.

Reply:
I was experimenting initially on a system tzdata (which have symlinks), but version in cctz - doesn't 

ls -la /usr/share/zoneinfo/posix